dtmi/stmicroelectronics/b_l4s5i_iot01a/standard_fw-1.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
please remove the brackets. Each file should contain a single interface

Reply:
OK, I have removed the brackets and pushed changes as requested.